docs/development.md

```diff
@@ -27,15 +27,27 @@ queries. After changing an sqlc `.sql` file, generate Go with:
 
 1. First, prepare a PR with a `CHANGELOG.md` update describing the changes, and update the root `go.mod` to point to the `riverpgxv5` version that is about to be released.
 2. Merge the above PR.
-3. Next, fetch the repo locally and push new tags:
+3. Next, fetch the repo locally, bump dependency versions, tag each submodule, and push those tags:
 
 ```shell
 git checkout master && git pull --rebase
-VERSION=v0.0.x
+export VERSION=v0.0.x
+go run ./internal/cmd/update-submodule-versions/main.go
 git tag cmd/river/$VERSION -m "release cmd/river/$VERSION"
 git tag riverdriver/$VERSION -m "release riverdriver/$VERSION"
 git tag riverdriver/riverpgxv5/$VERSION -m "release riverdriver/riverpgxv5/$VERSION"
 git tag riverdriver/riverdatabasesql/$VERSION -m "release riverdriver/riverdatabasesql/$VERSION"
 git tag $VERSION
 git push --tags
 ```
+
+### Releasing River CLI
+
+The CLI (`./cmd/river`) is different than other River submodules in that it doesn't use any `replace` directives so that it can stay installable with `go install ...@latest`.
+
+If changes to it don't require updates to its other River dependencies (i.e. they're internal to the CLI only), it can be released normally as shown above.
+
+If updates to River dependencies _are_ required, then a two-phase update is necessary:
+
+1. Release River dependencies with an initial version (e.g. `v0.0.14`).
+2. From `./cmd/river`, `go get` to upgrade to the version from (1), run `go mod tidy`, then release another version (e.g. `v0.0.15`).
```

Comment:
I think we can tweak this to avoid the double versioning as I did in my testing. All you have to do is tag & push everything else first as above but skip the `cmd/river` tag.

Once that's done, you can follow step 2 here to bump the versions in `cmd/river/go.sum` to the newly released version, commit & push that, then tag it as `cmd/river/v0.0.14`.

The released versions won't have the same commit, but the numbers will at least align this way.

Reply:
Cool. It feels a little weird to have the tags point to different commits, but I guess it doesn't matter. Updated the docs.

Reply:
Indeed. Both of these options are weird, thanks Go 😆